public/video-ui/src/components/FormFields/ItemPicker.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import KeywordPicker from '../utils/KeywordPicker';
+
+class ItemPicker extends React.Component {
```

Comment:
I think it would be clearer to make `ItemPicker` and `KeywordPicker` one component, unless it is re-used elsewhere